src/jvmMain/kotlin/io/github/fourlastor/data/Entity.kt

```diff
@@ -76,6 +77,11 @@ data class Image(
     override fun name(name: String) = copy(name = name)
 
     override fun collapsed(collapsed: Boolean) = copy(collapsed = collapsed)
+
+    fun rows(rows: Int) = copy(frame = frame.copy(rowsProperty = frame.rowsProperty.copy(value = rows.toFloat())))
+    fun columns(columns: Int) = copy(frame = frame.copy(columnsProperty = frame.columnsProperty.copy(value = columns.toFloat())))
+    fun frameNumber(frameNumber: Int) = copy(frame = frame.copy(frameNumberProperty = frame.frameNumberProperty.copy(value = frameNumber.toFloat())))
```

Comment:
minor: we could have `frame.rows/columns/frameNumber` to wrap the copy inside, so here it would be `fun rows(rows: Int) = copy(frame = frame.rows(rows))` and so on